claim-check/code-samples/sample-4/client-producer/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Configuration;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace client_producer
+{
+    class Program
+    {
+        public static PayloadDetails payloadDetails = new PayloadDetails();
+
+        public static void Main()
+        {
+            Console.WriteLine("Begin sample 4 for Claim n Check pattern");
+            Console.WriteLine();
+
+            Console.WriteLine("Creating payload and uploading it to Azure Blob Storage...");
+            ProcessAsync().GetAwaiter().GetResult();
```

Comment:
Use await when the above change is made.

Reply:
Done